src/QMCDrivers/VMC/VMCDriverInput.cpp

```diff
@@ -21,7 +21,7 @@ void VMCDriverInput::readXML(xmlNodePtr node)
 {
   ParameterSet parameter_set_;
   parameter_set_.add(walkers_per_rank_, "walkers", "int");
-  std::string use_drift;
+  std::string use_drift("yes");
```

Comment:
The manual says usedrift has a default of no.  So the default for VMCDriverInput::use_drift_ = false; and this string is not set. 
The goal is to have all default values expressed in the header with respect to there actual run time values.  Their defaults should not be set as they are expressed in a non-c++ input format. So if your intent is that the default change then we should change in the header `VMCDriverinput::use_drift_ = true` and in the readXML `if(use_drift == "no") use_drift_ = false;`

Reply:
I will fix this up in a my next PR.